src/components/SequenceCard/SequenceCard.stories.js

```diff
@@ -14,3 +14,14 @@ Default.args = {
     'Schuss Leipzig Pass von Sabitzer, Werner flach links am Tor vorbei',
   lengthTimeCode: '00:00:32:15',
 }
+
+export const WithTag = Template.bind({})
+WithTag.args = {
+  description:
+    'Goretzka chippt auf Müller, dessen Kopfballbogenlampe aufs Tordach fliegt.',
```

Comment:
I hope you're not a fan of that `miasanmia` club 🤣

Reply:
;-) no. was the first kicker ticker input i found.